src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.sh

```diff
@@ -0,0 +1,29 @@
+__PROJECT_DIR=$1
+__DOTNET_CMD=$2
+__TOOLRUNTIME_DIR=$3
+__TOOLS_DIR=$(cd "$(dirname "$0")"; pwd -P)
+
+__BUILDERRORLEVEL=0
+
+if [ ! -d "$__PROJECT_DIR" ]; then
+   echo "ERROR: Cannot find project root path at '$__PROJECT_DIR'. Please pass in the source directory as the 1st parameter."
+   exit 1
+fi
+
+if [ ! -d "$__TOOLS_DIR" ]; then
+   echo "ERROR: Cannot find tools path at '$__TOOLS_DIR'. Please pass in the tools directory as the 2nd parameter."
+   exit 1
+fi
+
+if [ ! -e "$__DOTNET_CMD" ]; then
+   echo "ERROR: Cannot find dotnet.exe at path '$__DOTNET_CMD'. Please pass in the path to dotnet.exe as the 3rd parameter."
+   exit 1
+fi
+
+$__DOTNET_CMD restore $__TOOLS_DIR/tool-runtime/project.json
+$__DOTNET_CMD publish $__TOOLS_DIR/tool-runtime/project.json -f dnxcore50 -r ubuntu.14.04-x64 -o $__TOOLRUNTIME_DIR
+cp -a $__TOOLS_DIR/. $__TOOLRUNTIME_DIR/
```

Comment:
Wow that hack wasn't at all obvious to me :) Seem like there should be a better way. I assumed that you put the cmd file into another directory beyond the lib folder.